src/components/shared/chat.tsx

```
@@ -1171,34 +1184,396 @@ export function CardsChat({
+                                    !message.content.match(
+                                      /\.(jpeg|jpg|gif|png|pdf|doc|docx|ppt|pptx)(\?|$)/i,
+                                    ) && (
+                                      <>
+                                        <div
+                                          className={cn(
+                                            'w-full break-words',
+                                            isEmojiOnly &&
+                                              'text-4xl leading-snug text-center',
+                                          )}
+                                          dangerouslySetInnerHTML={{
+                                            __html: DOMPurify.sanitize(
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)